arroyo/processing/strategies/run_task_with_multiprocessing.py

```diff
@@ -421,6 +458,7 @@ def __check_for_results(self, timeout: Optional[float] = None) -> None:
                 result.next_index_to_process = idx
 
         if result.next_index_to_process != len(input_batch):
+            self.__metrics.increment("batch.output.overflow")
```

Comment:
```suggestion
            self.__metrics.increment("arroyo.strategies.run_task_with_multiprocessing.batch.output.overflow")
```
It's a mouthful I know but I've started prefixing new metrics this way, and would like to migrate the existing ones at some point